### DISCUSSION TITLE
ImportError: no module named serial

### DISCUSSION THREAD

**Discussion by lnart** [jomjol/AI-on-the-edge-device#1171] [Q&A]
I started to flash my esp32 cam but when I try this error appears:

`Traceback (most recent call last):
  File "./esptool-3.3.2/esptool.py", line 27, in <module>
    import serial
`

I installed es-tool-3.3.2 and AI-on-the-edge-device-12.0.1 and I am in the right terminal where I executed the following code:

`./esptool-3.3.2/esptool.py \ --port /dev/cu.usbserial-10 write_flash \ 0x01000 ./AI-on-the-edge-device-12.0.1/firmware/bootloader.bin \ 0x08000 ./AI-on-the-edge-device-12.0.1/firmware/partitions.bin \ 0x10000 ./AI-on-the-edge-device-12.0.1/firmware/firmware.bin`

If anyone knows what I have to do or what the problem is pls hum

**Comment by ghost** [comment]
pip install serial?

**Reply by lnart** [reply]
I've already done that but I've git the same error when I try it afterwards

**Reply by ghost** [reply]
Wich python version you use? 

Looks like you installed esp-tool via download. Best way ist to install via pypi.

```pip install esp-tool```

I use python 3.9 and installed via pip without any issue on Mac.
